src/components/Cart/Steps/Steps.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="np-cart__steps__container">
+    <div v-for="(element,index) in steps" :key="index" class="np-cart__step">
+      <div v-if="stepValue - 1 === index" class="np-cart__step__circle--active"/>
+      <div v-if="stepValue - 1 > index" class="np-cart__step__circle--completed"/>
+      <div v-if="stepValue - 1 < index" class="np-cart__step__circle--waiting"/>
+      <p v-if="stepValue-1 > index" class="text-primary">{{ element }}</p>
+      <p v-if="stepValue-1 < index" class="text-secondary">{{ element }}</p>
+      <p v-if="stepValue-1 === index">{{ element }}</p>
+    </div>
+  </div>
+</template>
+
+<script>
+import {defineComponent} from "vue";
+import {mapGetters} from "vuex";
+
+const Steps = defineComponent({
+  data() {
+    return {
+      steps: ['My order', 'Information', 'Summary'],
```

Comment:
jezeli korzystanie z tsa to mozesz zerknac na
https://www.typescriptlang.org/docs/handbook/enums.html
kroki beda niezmienne, wiec mozna zrobic
`enum Steps { MY_ORDER = 'myOrder, INFORMATION = 'information', SUMMARY = 'summary' }`
a potem `steps: [ Steps.MY_ORDER, Steps.INFORMATION, Steps.SUMMARY ]`